core/amazon/index.js

```diff
@@ -323,8 +328,47 @@ var amazonProduct = {
         return Promise.all(promiseArray).then(() => {
             return result;
         });
+    },
+
+    isItemPrimeEligible: function (item, isVariationOffer) {
+        let offers = item.Offers && item.Offers[0];
+        if (!offers)
+            return false;
+
+        let isPrime = false;
```

Comment:
I'd prefer if you didn't mix styles here... either use the following if-else statement to set the isPrime variable which you always return at the end or just return true or false from inside the if-else statement whenever you want. Right now you use both methods